src/raftstore/store/bootstrap.rs

```diff
@@ -90,7 +92,9 @@ pub fn write_prepare_bootstrap(engines: &Engines, region: &metapb::Region) -> Re
     let raft_wb = WriteBatch::new();
     try!(write_initial_raft_state(&raft_wb, region.get_id()));
     try!(engines.kv_engine.write(wb));
+    try!(engines.kv_engine.flush_wal(true));
     try!(engines.raft_engine.write(raft_wb));
+    try!(engines.raft_engine.flush_wal(true));
```

Comment:
If you wanna sync wal in here, you can use write_opt instead.

Reply:
maybe just use sync_wal is more simple